concep_Timeword/conceptual.md

```diff
@@ -1,30 +1,52 @@
-### Conceptual Exercise
+## Conceptual Exercise
 
 Answer the following questions below:
 
-- What is a JWT?
+#### What is a JWT?
+* JSON Web Token - a compact URL-safe means of representing claims to be transferred between two parties.
+In other words, JWTs are credentials, which can grant access to resources.

```

Comment:
And since it's JSON, it typically contains user data so the server doesn't have to make another request to get data for the user the JWT was created for.